jha/src/views/enrolment/ContactInfoPage.vue

```diff
@@ -424,6 +434,20 @@ export default {
       this.isMailSame = false;
     },
   },
+  watch: {
+    mailCountry: function(value, oldValue) {
+      // don't clear province if the change is from loading from store
+      if (oldValue !== "_") {
```

Comment:
Just saw this. I don't think `mailCountry` will ever be `"_"`, as it gets overridden by the store value ("Canada"). You can probably remove this check. I think the initial value for `mailCountry` should be `null` anyway.

Reply:
No it is needed, if they navigate to a previous page and then return, then the country and province will be loaded from the store, if this check isn't there, then the watcher will trigger when it loads from store and overwrite the province field with either "British Columbia"(if stored country is Canada) or ""(if stored country is anything else).  Watchers are triggered when the value is changed from the default set in data during the created function.

Reply:
Ok, for that case. We usually use a `isPageLoaded` flag, and only make the assignments in the watch function when the flag is `true`.

Here's an example on the Personal Info page: https://github.com/bcgov/MOH-MSP-Enrolment/blob/main/jha/src/views/enrolment/PersonalInfoPage.vue#L860